### PR TITLE
Update to current version of flutter and ML Kit. Custom models extended

### PR DESCRIPTION
**Pull request by RichiWIP**
I have updated the code and dependencies to run with the current version of flutter and ML Kit. 
I have also updated the plugin to allow multiple inputs and outputs for custom models so that more complex models like object detection can be used, as well as adding an example app for this.
I am just begging to use flutter and have very little experience with it, as well as practically no experience with iOs / Objective-c.
I'd be happy for any suggested improvements.

### CODE REVIEW

**Comment by azihsoyn** [comment]
Hi, @RichiWIP!

Thanks to contributing and sorry to late responding.

I'll review within a few days.

File: example/custom-models/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+  ObjectDetectionLabel([this.label, this.confidence, this.box]);
+  ObjectDetectionLabel.box(this.box);
+}
+
+class _MyAppState extends State<MyApp> {
+  List<String> _models = ["mobilenet_quant", "mobilenet_float", "coco"];
+
+  File _file;
+  int _currentModel = 3;
```
Comment (azihsoyn) [inline example/custom-models/lib/main.dart]: this causes RangeError exception

File: example/custom-models/ios/Runner/GoogleService-Info.plist [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (azihsoyn) [inline example/custom-models/ios/Runner/GoogleService-Info.plist]: you should not your GoogleService-Info.plist for security.

**Comment by azihsoyn** [comment]
Thank you for adding example code for the custom model!

Currently, two flutter projects in the examples directory. It is just hard for maintenance.

If you can, please unite to one project like a [flutter_gallery app](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery).

File: example/custom-models/android/app/src/main/java/com/example/detect/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.example.detect
```
Comment (azihsoyn) [inline example/custom-models/android/app/src/main/java/com/example/detect/MainActivity.java]: need semicolon

**Comment by azihsoyn** [comment]
Hi, @RichiWIP.

I'm sorry conflicting by my [PR](https://github.com/azihsoyn/flutter_mlkit/pull/63) to resolve #59 and  #62 quickly.

Please merge master branch to reduce diff for reviewable.

Thanks.

File: example/custom-models/ios/Runner/GoogleService-Info.plist (hunk 2 above)
Comment (RichiWIP) [reply example/custom-models/ios/Runner/GoogleService-Info.plist]: Thanks, I thought I had deleted all my google files.

**Comment by azihsoyn** [comment]
Hi, @RichiWIP!

Thanks, great PR!

LGTM.

**Pull request merged**